agoradatatools/etl/transform.py

```diff
@@ -182,146 +164,105 @@ def transform_rna_seq_data(datasets: dict, adjusted_p_value_threshold: int):
-        ['ensembl_gene_id', 'name', 'summary', 'alias', 'igap', 'symbol', 'eqtl', 'rna_in_ad_brain_change',
-         'rna_brain_change_studied']]
-
-    proteomics_concat = pd.concat([proteomics, proteomics_tmt])
     
+    # Get the smallest cor_pval for each protein, to determine significance
+    proteomics_concat = pd.concat([proteomics, proteomics_tmt])
     proteomics_concat = proteomics_concat.dropna(subset=['log2_fc', 'cor_pval', 'ci_lwr', 'ci_upr'])
     proteomics_concat = proteomics_concat.groupby('ensg')['cor_pval'].agg('min').reset_index()
-
-    gene_metadata = pd.merge(left=gene_metadata,
-                             right=proteomics_concat,
-                             how='left',
-                             left_on='ensembl_gene_id',
-                             right_on='ensg')
-    gene_metadata['cor_pval'] = gene_metadata['cor_pval'].fillna(-1)
-    gene_metadata['protein_brain_change_studied'] = gene_metadata.apply(
-        lambda row: False if row['cor_pval'] == -1 else True, axis=1)
-    gene_metadata['protein_in_ad_brain_change'] = gene_metadata.apply(
-        lambda row: True if row['cor_pval'] <= protein_level_threshold else False, axis=1)
-
-    gene_metadata = gene_metadata[
-        ['ensembl_gene_id', 'name', 'summary', 'symbol', 'alias', 'igap', 'eqtl', 'rna_in_ad_brain_change',
-         'rna_brain_change_studied', 'protein_in_ad_brain_change', 'protein_brain_change_studied']]
-    gene_metadata.rename(columns={'symbol': 'hgnc_symbol'}, inplace=True)
-
-    return gene_metadata
-
-
-def transform_gene_info(datasets: dict):
-
-    gene_metadata = datasets['gene_metadata']
-    target_list = datasets['target_list']
-    median_expression = datasets['median_expression']
-    druggability = datasets['druggability']
-
+    proteomics_concat.rename(columns = {'ensg': 'ensembl_gene_id'}, 
+                             inplace = True)
+    
     # these are the interesting columns of the druggability dataset
     useful_columns = ['geneid', 'sm_druggability_bucket', 'safety_bucket', 'abability_bucket', 'pharos_class',
                       'classification', 'safety_bucket_definition', 'abability_bucket_definition']
     druggability = druggability[useful_columns]
 
     target_list = nest_fields(df=target_list,
                               grouping='ensembl_gene_id',
-                              new_column='nominated_target')
+                              new_column='nominatedtarget')
 
     median_expression = nest_fields(df=median_expression,
                                     grouping='ensembl_gene_id',
-                                    new_column='median_expression')
+                                    new_column='medianexpression')
 
     druggability = nest_fields(df=druggability,
                                grouping='geneid',
                                new_column='druggability')
     druggability.rename(columns={'geneid': 'ensembl_gene_id'}, inplace=True)
```

Comment:
another rename in code that should be handled in config

Reply:
Undid this change so naming is snake_case as before. (commit [1258ed40f91d6842e381cbaaac1eeef0d729c6cf](https://github.com/jaclynbeck-sage/agora-data-tools/commit/1258ed40f91d6842e381cbaaac1eeef0d729c6cf))

Reply:
To clarify: I changed `medianexpression` back to `median_expression`. I did not change the druggability rename. See [this JIRA task](https://sagebionetworks.jira.com/browse/AG-893) for why druggability columns currently can not be renamed using the config.